frontend/src/pages/dashboard/editor/Editor.tsx

```diff
@@ -135,6 +113,19 @@ export default function Editor(props: {}) {
 				<i className={ "fi fi-sr-brain icon-test" } style={ { fontSize: "2em" } }/>
 				<h3>{ t('dashboard.questionEditor.type.simp.title') }</h3>
 				<p>{ t('dashboard.questionEditor.type.simp.hint') }</p>
+				
+				<span className={ "caption" }>AI Model</span>
```

Comment:
Is this a dev component or is it for real user, if so please remember the requirement i18n